Content.Server/Botany/Systems/PlantHolderSystem.cs

```
@@ -885,13 +885,20 @@ public void UpdateReagents(EntityUid uid, PlantHolderComponent? component = null
 
         if (solution.Volume > 0 && component.MutationLevel < 25)
         {
-            foreach (var entry in component.SoilSolution.Value.Comp.Solution.Contents)
+            var quantity = FixedPoint2.New(1);
+
+            // Don't apply any effects to a non-unique seed ever! Remove this when botany code is sane...
+            EnsureUniqueSeed(uid, component);
+            foreach (var entry in solution.Contents)
             {
+                if (entry.Quantity < quantity)
+                    continue;
```

This removes the ability to affect the plant with a partial amount. Anything less than a whole unit gets silently discarded and doesn't affect the plant.

---

Currently no botanty effects use scale, so this just ensures you can't dilute your reagents. A proper solution needs a botany refactor.